homeassistant/components/trace/websocket_api.py

```diff
@@ -72,42 +70,45 @@ def websocket_automation_trace_get(hass, connection, msg):
 @callback
 @websocket_api.require_admin
 @websocket_api.websocket_command(
-    {vol.Required("type"): "automation/trace/list", vol.Optional("automation_id"): str}
+    {
+        vol.Required("type"): "trace/list",
+        vol.Inclusive("domain", "id"): vol.In(TRACE_DOMAINS),
+        vol.Inclusive("item_id", "id"): str,
```

Comment:
Why are these inclusive? I don't think I ever want to get all traces from both scrips and automation at the same time.